_includes/master/manifests/_functions.tpl

```diff
@@ -16,3 +16,12 @@ Canal
 Calico
 {{- end -}}
 {{- end -}}
+
+{{- define "etcd_tls" -}}
```

Comment:
I think "the helm way" is to prefix with your package name and use camelcase, so `calico.etcdTls`